docs/use-cases/deploy-ml.md

```diff
@@ -1,77 +1,114 @@
-weight: 50
+title: "How to train and deploy ML/computer vision models"
+linkTitle: "Train computer vision models"
+weight: 20
 type: "docs"
-tags: ["data management", "data", "services"]
-no_list: true
-description: "Use Viam's machine learning capabilities to train image classification models and deploy these models to your machines."
-images: ["/platform/ml.svg"]
-imageAlt: "Machine Learning"
+tags: ["vision", "data", "services"]
+images: ["/services/ml/collect.svg"]
+description: "Collect images and do interesting things with computer vision, ML, and webhooks."
 ---
 
-You can create and deploy an image classification model onto your machine with Viam's machine learning (ML) capabilities.
-Manage the classification model fully on one platform: collect data, create a dataset and label it, and train the model for **Single** or **Multi Label Classification**.
-Then, test if your model works for classifying objects in a camera stream or existing images with the `mlmodel` classification model of vision service.
+You can use Viam's built-in tools to train a machine learning (ML) model on your images and then deploy computer vision on your machines.
+
+![Diagram of the camera component to data management service to ML model service to vision service pipeline.](/use-cases/ml-vision-diagram.png)
+
+For example, you can train a model to recognize your dog and detect whether they are sitting or standing.
+Then, you can configure your machine to [capture images](/use-cases/image-data/) only when your dog is in the camera frame so you don't capture hundreds of photos of an empty room.
+You can then get even more image data of your dog and improve your ML model by training it on the larger dataset.
+
+You can do all of this using the [Viam app](https://app.viam.com) user interface.
+You will not need to write any code.
+
+<div style="border:1px solid; box-shadow:5px 5px 0 0;padding:1rem; margin:1rem; background-color:#f5f5f5">
+
+**In this page:**
+
+1. [Create a dataset and label data](#create-a-dataset-and-label-data)
+2. [Train and test a machine learning (ML) model](#train-and-test-a-machine-learning-ml-model)
+
+</div>
+
+## Create a dataset and label data
 
 {{< table >}}
 {{< tablestep >}}
-{{<imgproc src="/services/ml/collect.svg" class="fill alignright" style="max-width: 300px" declaredimensions=true alt="Collect data">}}
-**1. Collect**
+{{<imgproc src="/services/icons/data-capture.svg" class="fill alignleft" style="max-width: 150px" declaredimensions=true alt="Collect data">}}
+**1. Collect images**
+
+Start by collecting images from your cameras and syncing it to the [Viam app](https://app.viam.com).
+See [Collect image data and sync it to the cloud](/use-cases/image-data/#collect-image-data-and-sync-it-to-the-cloud) for instructions.
 
-Start by collecting images from your cameras with the [data management service](/services/data/).
-You can [view the data](/services/data/view/) on the **Data tab**.
+{{% alert title="Tip" color="tip" %}}
```

Comment:
Please move this under the steps for creating and labeling data. You might need to adjust the text a little but I think since it's more of an aside anyway it can go at the bottom of this segment

Reply:
Even though they'd need to do this during this step, before collecting the data?